client/web/web.go

```diff
@@ -790,7 +790,11 @@ func (s *Server) tailscaleUp(ctx context.Context, st *ipnstate.Status, opt tails
 
 	go func() {
 		if !isRunning {
-			s.lc.Start(ctx, ipn.Options{})
+			ipnOptions := ipn.Options{AuthKey: opt.AuthKey}
```

Comment:
These not applicable for the other paths?

Reply:
no, both of the other two have an existing registration and are simply reconnecting or reauthenticating.  I guess in the reauth case you might use an auth key (?), but I doubt that's common.  We can always add it later?